File: 30 Day Coding Challenge/1 October 2023.java
// Question 1

public class Solution {  

    public static int pairSum(int arr[], int x) {
    	//Your code goes here
        int count=0;
        for(int i =0;i<arr.length;i++){
            for(int j=i+1;j<arr.length;j++){
                if(arr[j] == x-arr[i])
                    count++;
            }
        }
        return count;
    }
}

File: 30 Day Coding Challenge/1 October 2023.py
# Question 1


from sys import stdin


def pairSum(arr, n, x) :
    #Your code goes here
    count =0;

    for i in range(n):
        for j in range(i+1,n):
            if arr[j] == x-arr[i]:
                count +=1
    
    return count;



